src/main/java/com/epam/repository/TourOfferDAO.java

```diff
@@ -34,7 +34,7 @@ public int addTour(TourOffer touroffer){
     }
 
     public int updateTour(TourOffer touroffer){
-       return JdbcTemplate.update("UPDATE tour_ofer SET " +
+       return JdbcTemplate.update("UPDATE tourOffer SET " +
```

Comment:
What's wrong with this name?

Reply:
In another pull request i fixed all name issues, but this branch was checked out from dev, where name conflicts still alive and there i fixed problems only in necessary methods